Homework_2,py

```diff
@@ -0,0 +1,44 @@
+# Homework_02
+
+# 1. create a list of random number of dicts (from 2 to 10)
+# dict's random numbers of keys should be letter,
+# dict's values should be a number (0-100),
+# example: [{'a': 5, 'b': 7, 'g': 11}, {'a': 3, 'c': 35, 'g': 42}]
+
+import random #import library for useing random function
+import string #import library for working with strings
+
+n = random.randint (2, 10) # length of list with dictionaries
+
+res_list=[] # empty list for adding disctionaries
+
+for i in range(n): # circle for the number of elements in list
```

Comment:
If you have time, try to use list comprehension. Not obligatory to fix

Reply:
Done